### ISSUE TITLE
Data Storage: Logs

### ISSUE THREAD

**Issue by fernandocbhorta** [ShannonHilland/T4TacToe#10]
a.	What is your security recommendation? Why did you choose it?
I've chosen "No sensitive data is written to application logs" as security recommendation to discuss. Before we submit the full project there were a few messages output to console via that could lead to problems with app security in the future.

b.	Who does the recommendation benefit (end-user, developer, etc.)?
Taking the log messages out of console benefits the end-user, knowing their information is no accessible from an easy way

c.	If the recommendation was found somewhere other than the provided checklist, include a link to it. 
Not applicable.

d.	When would the recommendation have to be implemented (based on how serious the security risk is)?
We've implemented this change right away, before submission, based on instructor's feedback.

e.	Why do you think your project needs your recommendation?
Hiding data, specially when it mentions table names and columns, is always a good idea - even though the app is on GitHub on a public repo

f.	How do you think your recommendation could be applied?
Basically removing the console.log lines or commenting them out. Adding a proper class to handle logs and, perhaps, store them with sensitive data encrypted on database would be a good idea.

i.	How feasible would the implementation be?
It would probably take some time to develop a proper log class, but definitely doable.
